modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -119,9 +119,9 @@ NOTE: Terms and conditions are only available for API assets. Users are prompted
 
 === API Groups
 
-If any API in an API group has a terms and conditions page, then the API group has a terms and conditions page that cannot be deleted and that shows the terms and conditions of the underlying APIs.
+If any API in an API Group has a terms and conditions page, then the API Group has a terms and conditions page that cannot be deleted and that shows the terms and conditions of the underlying APIs.
```

Comment:
An API Group displays a terms and conditions page that cannot be deleted, based on whether the API in that group displays such a page?